content/en/docs/contribute/_index.md

```diff
@@ -4,59 +4,76 @@ title: Contribute to Kubernetes docs
 linktitle: Contribute
 main_menu: true
 weight: 80
+card:
+  name: contribute
+  weight: 10
+  title: Start contributing
 ---
 
 {{% capture overview %}}
 
-If you would like to help contribute to the Kubernetes documentation or website,
-we're happy to have your help! Anyone can contribute, whether you're new to the
-project or you've been around a long time, and whether you self-identify as a
-developer, an end user, or someone who just can't stand seeing typos.
+This website is maintained by [Kubernetes SIG Docs](/docs/contribute/participating/).
```

Comment:
nit: Why does this sentence link to the participating page and not to the Sig-Docs section on this page,
https://deploy-preview-19576--kubernetes-io-master-staging.netlify.com/docs/contribute/#get-involved-with-sig-docs